src/main/java/net/glowstone/block/blocktype/BlockPumpkinBase.java

```diff
@@ -0,0 +1,30 @@
+package net.glowstone.block.blocktype;
+
+import net.glowstone.block.GlowBlockState;
+import net.glowstone.entity.GlowPlayer;
+import net.glowstone.inventory.ToolType;
+import org.bukkit.Material;
+import org.bukkit.block.BlockFace;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.material.MaterialData;
+import org.bukkit.material.Pumpkin;
+import org.bukkit.util.Vector;
+
+public abstract class BlockPumpkinBase extends BlockDirectDrops {
```

Comment:
Removed abstract (was needed in an earlier iteration when I was changing the constructors but no longer necessary)